python/cudf/cudf/comm/serialize.py

```diff
@@ -1,59 +1,31 @@
-import pickle
-
-import cudf
-import cudf.core.groupby.groupby
-
-# all (de-)serializations are attached to cudf Objects:
-# Series/DataFrame/Index/Column/Buffer/etc
-serializable_classes = (
```

Comment:
+1 one from me as well! I especially like moving away from this list to anything which implements Serializable